.eslintrc.js

```diff
@@ -0,0 +1,48 @@
+module.exports = {
```

Comment:
Because it's easier to read & edit JS than JSON. 

But if JS bother you, what about using YAML instead, like GitHub Actions?

Reply:
FYI `jest.config.js` is also a JS file.

Reply:
It does not bother me to use JS instead of JSON, however, I usually prefer using JSON instead of JS when I have no need to run some code in it (which I usually don’t).

As for YAML, I still prefer JSON. :wink:

As for readability, for me it is same if I read a JS object or a JSON, as long as I read them _both_ with nice syntax highlighting. As for editting, it is the same to me, as long as I have configured my text editor to use JSON schemas (for JSONs), I don’t need to really worry about double quotes (the editor auto-inserts them when I select a key from auto-suggestions even when I type the key name without quotes).

For for `jest.config.js` being a JS file, yes, it is, however, you didn’t modified it in the PR. However, I’d still prefer using JSON here too, unless we need to run some JS code in there.

Nevertheless, we can keep these files in JS. :wink: